### PR TITLE
initial install of material ui

### PR DESCRIPTION
**Pull request by BeeSeeWhy**
- initial install of material ui
- initial install of roboto font (material ui utilizes this font)
- fix 3 high severity vulnerabilities via npm audit fix

### CODE REVIEW

**Comment by BeeSeeWhy** [comment]
I think I know why this check is failing....fixing

**Comment by amantri** [comment]
> I think I know why this check is failing....fixing

I had narrowed down this failure to `npm audit fix`. Didn't get a chance to find a fix yet.

**Comment by codetheweb** [comment]
I think you'll probably want to set up a theme provider as well, e.x. https://medium.com/frontendweb/how-to-use-material-ui-with-nextjs-and-react-18-6c054ceacf77

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -25,6 +32,7 @@ export default function App({
   Component,
   pageProps,
 }: AppProps<CustomPageProps>) {
+  const emotionCache = clientSideEmotionCache;
```
Comment (codetheweb) [inline src/pages/_app.tsx:35]:
```suggestion
```

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -37,12 +45,18 @@ export default function App({
...
 
   return (
-    <trpc.Provider client={trpcClient} queryClient={queryClient}>
-      <QueryClientProvider client={queryClient}>
-        <SessionProvider session={pageProps.session}>
-          <Component {...pageProps} />
-        </SessionProvider>
-      </QueryClientProvider>
-    </trpc.Provider>
+    <CacheProvider value={emotionCache}>
```
Comment (codetheweb) [inline src/pages/_app.tsx:48]:
```suggestion
    <CacheProvider value={clientSideEmotionCache}>
```

**Review by codetheweb — APPROVED** [review]
looks good after the merge conflict is resolved!

**Comment by BeeSeeWhy** [comment]
moot issue, closing

**Pull request closed without merging**